ui/app/router.js

```diff
@@ -154,6 +154,7 @@ Router.map(function () {
         this.route('backends', { path: '/' });
         this.route('backend', { path: '/:backend' }, function () {
           this.mount('kmip');
+          this.mount('kubernetes');
```

Comment:
While this is in development, should we put this within the check for config.environment like the pki engine?